1-mandatory/4-budgets.js

```diff
@@ -10,7 +10,13 @@ getBudgets([
 Should give return the answer of 62600.
 **/
 
-function getBudgets(peopleArray) {}
+function getBudgets(peopleArray) {
+  let sumBudgets = 0;
+  for (let i = 0; i <peopleArray.length; i++) {
+    sumBudgets += peopleArray[i].budget
+  }
```

Comment:
this works fine, but would be better to use `.reduce` method here. 

Let me know if you have any questions